installing/installing_with_agent_based_installer/installing-with-agent-based-installer.adoc

```diff
@@ -71,6 +71,9 @@ include::modules/installing-ocp-agent-encrypt.adoc[leveloffset=+2]
 // Creating and booting the agent image
 include::modules/installing-ocp-agent-boot.adoc[leveloffset=+2]
 
+// Adding {ibm-z-name} agents with {op-system-base} KVM
+include::modules/installing-ocp-agent-ibm-z-kvm.adoc[leveloffset=+3]
```

Comment:
Right now with a level offset of 3, structurally the section now seems like a subtask of the "Creating and booting the agent image" procedure. If it should be considered that way then this can stay as is. But if this section documents a separate and not related action, this should have a level offset of 2, like this:
```suggestion
include::modules/installing-ocp-agent-ibm-z-kvm.adoc[leveloffset=+2]
```